packages/flutter/lib/src/rendering/object.dart

```diff
@@ -1367,6 +1408,10 @@ abstract class RenderObject extends AbstractNode with DiagnosticableTreeMixin im
   bool get _debugCanPerformMutations {
     late bool result;
     assert(() {
+      if (_debugDisposed) {
```

Comment:
Shouldn't asking to perform mutations on a disposed object just be illegal (i.e. fire an assert)?